krping.c

```diff
@@ -59,7 +59,7 @@
 
 #define PFX "krping: "
 
-static int debug = 0;
+static int debug = 1;
```

Comment:
Please remove this; no need to default debug to on.

Reply:
Ops, forgot changing it back..